README.md

```diff
@@ -10,9 +10,9 @@ LORIS is a web-accessible database solution for neuroimaging, providing a secure
  * LINUX (supported on Ubuntu 14.04 and CentOS 6.5) or Mac OS X (tested for Mavericks - OS X 10.9)
  * Apache2 (libapache2-mod-php5)
  * MySQL (libmysqlclient15-dev mysql-client mysql-server)
- * PHP/PEAR 5.3+ (php5 php-pear php5-mysql php5-gd)
+ * PHP 5.3+ (php5 php5-mysql php5-gd php5-sqlite)
+ * PEAR (php-pear)
```

Comment:
I don't think it should be there in the composer.json (I think it was required for the old SpreadsheetWriter?), but I agree that it should be a different pull request and not mixed into this unrelated one (also our official policy is going to be PHP 5.6+ for this release, so that needs to be updated separately..)